webapp/src/shared/types.ts

```diff
@@ -28,3 +30,12 @@ export type Avatar = {
   avatarX: number;
   avatarY: number;
 };
+
+type PlayState = 'playing' | 'paused';
```

Comment:
I think this should be already in pong engine, and imported from there, since we also have it and use it in game.service.ts